frontend/src/scenes/links/LinkScene.tsx

```diff
@@ -0,0 +1,131 @@
+                        {buttonText}
+                    </LemonButton>
+                }
+            />
+
+            <div className="space-y-4">
+                <div className="flex gap-8">
+                    <div className="flex-1 space-y-6">
+                        <LemonField name="redirect_url" label="Destination URL">
+                            <LemonInput placeholder="https://posthog.com/links" fullWidth autoWidth={false} />
+                        </LemonField>
+
+                        <div className="flex flex-col gap-2">
+                            <LemonLabel>Link</LemonLabel>
+                            <div className="flex gap-2">
+                                <LemonField name="short_link_domain">
+                                    <LemonSelect
+                                        disabledReason="More domains coming soon"
+                                        options={[
+                                            { label: 'postho.gg', value: 'postho.gg/' },
+                                            { label: 'phog.gg', value: 'phog.gg/' },
+                                            { label: 'hog.gg', value: 'hog.gg/' },
+                                        ]}
+                                        className="text-muted"
+                                    />
+                                </LemonField>
+                                <LemonField name="short_code" className="w-full">
+                                    <LemonInput
+                                        fullWidth
+                                        placeholder="(optional)"
+                                        className="flex-1"
+                                        autoWidth={false}
+                                    />
+                                </LemonField>
+                            </div>
+                        </div>
+
+                        <LemonField name="description" label="Description">
+                            <LemonTextArea placeholder="Add description" minRows={2} />
+                        </LemonField>
+                    </div>
+
+                    <LemonDivider vertical />
+
+                    <div className="flex-1 space-y-6 max-w-80">
+                        <div>
+                            <div className="flex justify-between items-center">
+                                <LemonLabel>
+                                    <span className="flex items-center gap-1">QR Code</span>
+                                </LemonLabel>
+                                <div className="flex flex-row">
+                                    <LemonButton
+                                        icon={<IconDownload />}
+                                        size="xsmall"
+                                        onClick={() => {}}
+                                        tooltip="Download QR code"
+                                    />
```

Comment:
Not downloading

(maybe note that in the UI, disable the button or something with a tooltip explaining it's coming)